templates/base.html

```diff
@@ -7,7 +7,11 @@
 
         <title>{{ block "title" . }}Orange Forum{{ end }}</title>
 
+        {{ if .Domain.Icon }}
+        <link rel="shortcut icon" href="{{ .Domain.Icon }}?v=223">
```

Comment:
The `?v=223` should be removed from here.

Reply:
Also, in the Domain struct in models/domains.go, the type of Icon should be changed to `template.URL` (just like logo).

Reply:
And can you also rebase all the commits to just one "Support per-domain favicon" commit and force push? Thx